lib/install/flow.js

```diff
@@ -41,8 +41,6 @@ module.exports = class Flow extends BaseStep {
     return step
     .start(this.install.state, this.install)
     .catch((err) => {
-      if (err.data) { this.install.updateState(err.data); }
```

Comment:
This line is problematic because failed requests from `kite-connect` would propagate up as part of `err.data`, resulting in a circular object error when trying to serialize to JSON:

![image](https://user-images.githubusercontent.com/3752225/47539670-50f12b80-d886-11e8-863d-a43d3eb6d7ee.png)